controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -408,3 +547,60 @@ void wfa_ca::handle_wfa_ca_message(
     }
     }
 }
+
+bool wfa_ca::create_cmdu(ieee1905_1::CmduMessageTx &cmdu_tx, ieee1905_1::eMessageType message_type)
+{
+    std::shared_ptr<ieee1905_1::cCmduHeader> cmdu_header;
+
+    int tlv_type;
+
+    switch (ieee1905_1::eMessageType(message_type)) {
```

Comment:
message_type is already an eMessageType.

Reply:
thanks. fixed.